samples/Server/src/lib/chat/moderator.ts

```diff
@@ -4,7 +4,6 @@
 import { AzureCommunicationTokenCredential } from '@azure/communication-common';
 import { ChatClient, CreateChatThreadOptions, CreateChatThreadRequest } from '@azure/communication-chat';
 import { getEndpoint } from '../envHelper';
-import { GUID_FOR_INITIAL_TOPIC_NAME } from '../constants';
```

Comment:
Can we remove this everywhere in the codebase? It shouldn't be being used anymore anywhere

Reply:
Updated :)